### PR TITLE
Updated four rasters and colourmaps to improve contrast

### PR DESCRIPTION
**Pull request by benkt2**
Also some description edits from KL

### CODE REVIEW

File: app/javascript/components/Map.vue [hunk 1]
```diff
@@ -154,27 +155,46 @@ export default {
...
-            "fill-color": carto.colour
-              ? carto.colour
-              : `#${(Math.random().toString(16) + "000000").substring(2, 8)}`,
-            "fill-opacity": 0.5
-          },
-          layout: {
-            visibility: visible ? "visible" : "none"
-          }
-        });
+	  console.log(carto.id)
```
Comment (WKocur) [inline app/javascript/components/Map.vue]: Console log :o

Comment (benkt2) [reply app/javascript/components/Map.vue]: oh, yes thats in the hacky bit that shouldn't be merged - proper fix incoming....

File: app/javascript/components/Map.vue [hunk 2]
```diff
@@ -154,27 +155,46 @@ export default {
...
+	  // Hackety Hack warning:
+	  if( carto.id.includes('River Quality')){
```
Comment (WKocur) [inline app/javascript/components/Map.vue]: Includes isn't supported by IE (you can use indexOf >= 0 (double check that, but I think it returns -1 if the substring isn't found).

File: app/javascript/components/Map.vue (hunk 1 above)
Comment (WKocur) [reply app/javascript/components/Map.vue]: Ah ok - in which case, you can ignore the next comment as well, as I assume that will change.

**Comment by WKocur** [comment]
All looks good to me - do you want the new seeds merged in, or are they good to wait for now?

**Pull request merged**